ListableUI/Sources/Item/ItemContent.swift

```diff
@@ -353,7 +353,10 @@ public protocol ItemContent where Coordinator.ItemContentType == Self
     /// The view type to use to render swipe actions (delete, etc) for this content.
     /// A default implementation, which matches `UITableView`, is provided.
     associatedtype SwipeActionsView: ItemContentSwipeActionsView = DefaultSwipeActionsView
-    
+
+    /// Specify a swipe action style for this content.
+    var swipeActionStyle: SwipeActionsView.Style { get }
```

Comment:
Nit: we're styling all the actions

```suggestion
    var swipeActionsStyle: SwipeActionsView.Style { get }
```